### ISSUE TITLE
Output WR scan report in other formats (e.g. json)

### ISSUE THREAD

**Issue by MaximMoinat** [OHDSI/WhiteRabbit#166]
Instead of the huge Excel sheet, we can implement other ways to export the scan report. An alternative could be json. Adantage over Excel is that it allows nested data structures. It can be easily viewed and navigated by text editors with json extensions. Also, json can serve as transfer format to other tools to display the data from the scan report.

**Comment by PRijnbeek** [comment]
Yes you can but what problem does it really solve? (see also #167).

Do we expect other tools to read the White Rabbit output? Why can they not read the Excel (really question is do you miss something in the Excel i think)

**Comment by MaximMoinat** [comment]
From my experience this solves just one minor issue on the short term. In tables with many columns, navigation through the excel sheet can be troublesome. Especially navigating from a variable in the Overview tab to the value list in the table tab.

RiaH actually does a good job at this visualisation as well, although you do need to define a mapping first which is not intuitive in the 'exploration' phase.

I will merge this with #167 in a more general enhancement; improving scan report navigation and reporting, preferably using RiaH.

**Comment by don-torok** [comment]
With regard to navigating WR scan reports I wrote some excel procedures that create links from the table name to the table detail and then the column name to the details for the column.  See https://github.com/OHDSI/sandbox/tree/master/WR_Scan_Formatter

**Comment by MaximMoinat** [comment]
@don-torok That is awesome! Thanks for sharing, can't wait to try this out.

Would be great to ingrate this by default in the WR output. From a quick search, the easiest option seems [to use a template Excel workbook](https://poi.apache.org/apidocs/dev/org/apache/poi/xssf/streaming/SXSSFWorkbook.html#SXSSFWorkbook-org.apache.poi.xssf.usermodel.XSSFWorkbook-)when creating the Scan Report.
